Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP_test.py

```diff
@@ -0,0 +1,1267 @@
+
+def test_computer_delete_error_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Delete a computer.
+    Given:
+    -   The user has entered a connector_guid.
+    When:
+    -   cisco-amp-computer-delete is called.
+    Then:
+    -   Ensure a value error has been raised.
+    """
+    args: Dict[str, Any] = {
+        'connector_guid': 1
+    }
+
+    mock_response = load_mock_response('computer_delete_fail_response.json')
+    requests_mock.delete(
+        f'{BASE_URL}/computers/{args["connector_guid"]}',
+        json=mock_response
+    )
+
+    from CommonServerPython import DemistoException
+    with pytest.raises(DemistoException) as de:
+        from CiscoAMP import computer_delete_command
+        computer_delete_command(mock_client, args)
+
+        assert de.message.startswith('Failed to delete Connector GUID:')
+
+
+def test_computer_activity_list_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get activity on computers by query.
+    Given:
+    -   The user has entered a url to query.
+    When:
+    -    cisco-amp-computer-activity-list is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure a links doesn't exist in outputs.
+    """
+    args = {
+        'query_string': '8.8.8.8'
+    }
+
+    mock_response = load_mock_response('computer_activity_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/computers/activity',
+        json=mock_response
+    )
+
+    from CiscoAMP import computer_activity_list_command
+    response = computer_activity_list_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerActivity'
+    assert_output_has_no_links(response.outputs)
+
+    for output, mock_output in zip(response.outputs, mock_response['data']):
+        assert output['connector_guid'] == mock_output['connector_guid']
+        assert output['hostname'] == mock_output['hostname']
+        assert output['active'] == mock_output['active']
+
+
+def test_computer_activity_list_error_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get activity on computers by query.
+    Given:
+    -   The user has entered a false query.
+    When:
+    -    cisco-amp-computer-activity-list is called.
+    Then:
+    -   Ensure a value has been raised.
+    """
+    args = {
+        'query_string': '"'
+    }
+
+    requests_mock.get(
+        f'{BASE_URL}/computers/activity'
+    )
+
+    with pytest.raises(ValueError) as ve:
+        from CiscoAMP import computer_activity_list_command
+        computer_activity_list_command(mock_client, args)
+
+        assert str(ve) == 'query_string must be: SHA-256/IPv4/URL/Filename'
+
+
+def test_computer_isolation_feature_availability_get_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get available features on a computer.
+    When:
+    -    cisco-amp-computer_isolation_feature_availability_get is called.
+    Then:
+    -   Ensure readable_output is correct.
```

Comment:
why only test for an empty output? What other outputs are expected?

Reply:
I've added headers to test the string is printed correctly.